pkg/cloudprovider/providers/gce/gce_backendservice.go

```diff
@@ -62,8 +62,8 @@ func (gce *GCECloud) DeleteBackendService(name string) error {
 	return gce.waitForGlobalOp(op, mc)
 }
 
-// CreateBackendService creates the given BackendService.
-func (gce *GCECloud) CreateBackendService(bg *compute.BackendService) error {
+// CreateGlobalBackendService creates the given BackendService.
```

Comment:
This could have been broken into several small and easy commits.  For future - please try to make each commit do exactly one thing.  Way faster to review.